PYIC-3117 Added postMitigations lamdba with test.

rpayal opened the pull request:
<!-- Provide a general summary of your changes in the Title above -->
<!-- Include the Jira ticket number in square brackets as prefix, eg `[P4-XXXX] PR Title` -->

## Proposed changes
Add postMitigations stub lamdba. 

### What changed
Added postMitigations lamdba with test. 

<!-- Describe the changes in detail - the "what"-->

### Why did it change

<!-- Describe the reason these changes were made - the "why" -->

### Issue tracking
<!-- List any related Jira tickets or GitHub issues -->
<!-- List any related ADRs or RFCs -->
<!-- Delete/copy as appropriate -->

- [PYI-3117](https://govukverify.atlassian.net/browse/PYIC-3117)

## Checklists

### Environment variables or secrets

<!-- Delete if changes DO include new environment variables or secrets -->
- [ ] No environment variables or secrets were added or changed

<!-- Delete if changes DO NOT include new environment variables or secrets -->
- [ ] Documented in the [README](./blob/main/README.md)
- [ ] Added to deployment repository
- [ ] Added to local startup repository

### Other considerations

- [ ] Update [README](./blob/main/README.md) with any new instructions or tasks

jkunle committed:
OJ-1702: Add deployment template for toy

jkunle committed:
OJ-1702: Add git action to deploy toy-cri

rpayal committed:
PYIC-3117 Added postMitigations lamdba with test. Also updated test and gradle config.

TalatAydinGDS commented on di-ipv-cimit-stub/deploy/template.yaml:
```diff
@@ -49,13 +49,48 @@ Resources:
+      FunctionName: !Sub "postMitigations-${Environment}"
+      CodeUri: "../lambdas/post-mitigations"
+      Handler: uk.gov.di.ipv.core.postmitigations.PostMitigationsHandler::handleRequest
+      Runtime: java17
```
could see java 11 in the build.gradle but here java 17.

therealslimsladey replied:
Java 17 Lambdas still kick out some warnings so safer to stick with Java 11 for now, methinks?

TalatAydinGDS replied:
I think, should be use to the same version java11 for Lambda

rpayal replied:
Done.

rpayal committed:
PYIC-3117 Updated lambda runtime to java11.

therealslimsladey committed:
PYIC-3114: Temporarily list build container build dir content for diagnosis

therealslimsladey committed:
Revert "PYIC-3114: Temporarily list build container build dir content for diagnosis"

rpayal committed:
Revert "PYIC-3114 Added new cimit stub base proj. for cimit stub lambda's, added getContraIndicators lamdba with test."

therealslimsladey committed:
BAU: Bump SAM CLI Github action version

rpayal committed:
PYIC-3114 Added new cimit stub base proj. for cimit stub lambda's, added getContraIndicators lamdba with test.

vam-gds committed:
PYIC-3238 create-a-public-jwk stubs account migration

github action and cloudformation deployment

	new file:   .github/workflows/public-jwk-creator.yml
	new file:   public-jwk-creator/deploy/template.yaml

rpayal committed:
PYIC-3114 Updated lambda memory and tracing.

rpayal committed:
PYIC-3114 Updated gradle-wrapper.jar.

rpayal committed:
PYIC-3114 Removed startup.sh not needed for this module.

rpayal committed:
PYIC-3114 Corrected spoyless

rpayal committed:
PYIC-3117 Updated lambda memory and tracing.
